internal/compiler/widgets/fluent/std-widgets.slint

```diff
@@ -377,7 +377,8 @@ export ListView := ScrollView {
 
 export StandardListView := ListView {
     property<[StandardListViewItem]> model;
-    property<int> current-item: -1;
+    property<int> current-item: min(actual-current-item, model.length - 1);
+    property<int> actual-current-item: -1;
```

Comment:
I wish we had a way to declare properties as private (Because i don't want user to rely on that.)
I guess it's fine as it is not documented (although the auto completion will auto complete.)
Maybe you can move that property in the FocusScope so it is inaccessible from the outside.

Reply:
done. but now the FocusScope has a name...